### PR TITLE
fix(stark-ui): fix side navigation rendering behind header

### PR DESCRIPTION
**Pull request by carlo-nomes**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/NationalBankBelgium/stark/blob/master/CONTRIBUTING.md#-commit-message-guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Because of a bug previously introduced in a previous commit the side navigations (left / right) were rendering behind the header, causing issues on mobile or whenever the side navigation has full page height.

## What is the new behavior?
This commit fixes the initial Safari bug (#1338) without breaking the side navigation.

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by nicanac** [comment]
Works well on my local.
But I noticed an issue with the travis' build.
Maybe a simple commit amend is enough.

**Comment by christophercr** [comment]
Can you guys double check in Firefox? In my case (on Mac) it seems is not possible to scroll anymore on the screens that are bigger than the viewport, for example the `demo-ui/input-mask-directives`page

File: packages/stark-ui/assets/styles/_base.scss [hunk 1]
```diff
@@ -17,15 +17,29 @@ body {
     margin-bottom: 64px;
   }
 
-  main.stark-main-container {
-    flex: 1 0 auto;
-    align-self: center;
+  .mat-drawer-content {
+    overflow: visible;
```
Comment (carlo-nomes) [inline packages/stark-ui/assets/styles/_base.scss]: @christophercr you are right, this line makes Firefox not respect the `height: 100%` of `.mat-drawer-content` (making it impossible to scroll in the `.stark-scroll-container`). The problem is... if I put set this to `overflow: hidden` Safari does not show the `<header>`. Do you or @nicanac or @SuperITMan  have any suggestion on the best way to resolve this? 🤔

**Comment by carlo-nomes** [comment]
Okay, I think I found / fixed the source of the problem. That was a difficult one. 😅Could you review again?

**Pull request merged**